### PR TITLE
Show duplicate users trusted federated server

### PR DESCRIPTION
**Pull request by mRoRo**

### CODE REVIEW

File: Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m [hunk 1]
```diff
@@ -207,16 +207,18 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
...
             
-            if (userOrGroup.shareeType == shareTypeRemote && userOrGroup.server != nil ) {
+            if (userOrGroup.shareeType == shareTypeRemote && userOrGroup.server != nil) {
                 
-                name = [NSString stringWithFormat:@"%@ (%@ %@)", userOrGroup.displayName, NSLocalizedString(@"at_federate_server", nil), userOrGroup.server];
-                
-            }else if(userOrGroup.isDisplayNameDuplicated){
-                
-                name = [NSString stringWithFormat:@"%@ (%@)", userOrGroup.displayName, userOrGroup.name];
+                name = (userOrGroup.isDisplayNameDuplicated)?
```
Comment (nasli) [inline Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m]: More readable with if ;)

File: Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m [hunk 2]
```diff
@@ -207,16 +207,18 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
...
+                [NSString stringWithFormat:@"%@ (%@)", userOrGroup.displayName, userOrGroup.name]:
+                [NSString stringWithFormat:@"%@ (%@)", userOrGroup.displayName, userOrGroup.server];
             }
+            
             else{
-                name = userOrGroup.displayName;
+                
+                name = (userOrGroup.isDisplayNameDuplicated)?
```
Comment (nasli) [inline Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m]: Same here, if

File: Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m [hunk 3]
```diff
@@ -437,21 +439,22 @@ - (void)customizeMessageView:(TSMessageView *)messageView
...
-    
-    OCShareUser *federatedUser = [OCShareUser new];
-    federatedUser.shareeType = shareTypeRemote;
-    federatedUser.isDisplayNameDuplicated = NO;
-    federatedUser.name = name;
-    federatedUser.displayName = [NSString stringWithFormat:@"%@ (%@)",name, NSLocalizedString(@"share_user_federated_indicator", nil)];
-    
-    return federatedUser;
-}
+//- (OCShareUser *) getFederatedOCSharedUserByName:(NSString *) name {
```
Comment (nasli) [inline Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m]: If not needed remove it

File: Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m (hunk 2 above)
Comment (mRoRo) [inline Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m]: And, here too...

File: Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m (hunk 3 above)
Comment (mRoRo) [inline Owncloud iOs Client/Tabs/FileTab/Share/ShareSearchUserViewController.m]: Sure. You're right

**Pull request merged**